### PR TITLE
Add a button to apply for volunteer positions

### PR DESCRIPTION
**Pull request by alberto-antonietti**

### CODE REVIEW

File: src/pages/join/volunteering.js [hunk 1]
```diff
@@ -31,11 +31,23 @@ TopicHeading.propTypes = {
...
+  applyButton.textContent = "Apply here for open volunteering positions";
+
+  // Set the button link
+  applyButton.onclick = function() {
+    window.location.href = "https://airtable.com/shrlo3trJbLCvqPmA";
+  };
+
   return (
     <Layout>
       <MainBlock>
```
Comment (iamzoltan) [inline src/pages/join/volunteering.js:47]: Looks good, does this button match the style of our other button? (see job board)
Does it change style when in dark-mode vs light mode?

**Review by iamzoltan — CHANGES_REQUESTED** [review]
Functionality looks good, lets make sure the style matches and there is a dark-mode vs light-mode button

**Comment by iamzoltan** [comment]
I am also getting this error when I run it locally:

```
document is not defined
  33 |
  34 |   // Create a button element
> 35 |   const applyButton = document.createElement("button");
     |                       ^
  36 |
  37 |   // Set the button text
  38 |   applyButton.textContent = "Apply here for open volunteering positions";
```

**Comment by alberto-antonietti** [comment]
@dalia-nasr would you like to handle this? My JS skills are very limited as you can see from above :)

**Pull request closed without merging**